rclpy/src/rclpy/_rclpy.c

```diff
@@ -43,6 +43,7 @@
 static PyObject * RCLInvalidROSArgsError;
 static PyObject * UnknownROSArgsError;
 static PyObject * NodeNameNonExistentError;
+static PyObject * RCLError;
```

Comment:
@suab321321 these changes seem to be outside the scope of this pull request, no? I think this should be filed separately.

Reply:
@allenh1 yes sirActually I was contributing to Create custom exceptions in _rclpy.c #31...
but when I did push it automatically pushed to this PR..
Sorry for this sir..I m new to opensource and dont know version control so well.

Reply:
@allenh1 sir I enough mess is created by me now....Should I just make new cloned repo and transfer my whole file from current to newone and then create a new PR..what does this sound sir?

Reply:
>  Sorry for this sir..I m new to opensource and dont know version control so well.

Not a problem, we can just clean it up.

> hould I just make new cloned repo and transfer my whole file from current to newone and then create a new PR..what does this sound sir?

I personally would prefer you not. It's better practice to just drop that commit or revert the commit that changed this file.

Opening a new PR makes it difficult to see what was already suggested (much like force pushing), and make sure that all points get resolved before something gets merged.